client/src/pages/CreateApplicationPage/CreateApplicationPage.tsx

```diff
@@ -159,8 +163,8 @@ const CreateApplicationPage = (): JSX.Element => {
             className="w-full p-2 rounded bg-gray-800 text-white"
             id="date_applied"
             name="date_applied"
-            type="date"
-            value={formData.date_applied}
+            type="datetime-local"
```

Comment:
including the time might be overkill

Reply:
Definitely. It is like this so that in the current environment I can test functionally without waiting days. I need the seconds for now.

Reply:
sounds good. should we prevent people from choosing a date/time in the future?

Reply:
Yes we should! New code inc.